src/domain/model/address.model.ts

```diff
@@ -0,0 +1,24 @@
+export interface AddressModel {
+  id: number;
+  userId: number;
+  cep: string;
+  street: string;
+  streetNumber: string;
+  complement?: string | null;
+  neighborhood: string;
+  city: string;
+  state: string;
+}
+
+export interface AddressInputModel {
+  data: {
+    userId: number;
+    cep: string;
+    street: string;
+    streetNumber: string;
+    complement?: string | null;
+    neighborhood: string;
+    city: string;
+    state: string;
+  };
+}
```

Comment:
pq o input fica dentro de um objeto chamada data? não daria pra interface ser os campos diretamente?
```suggestion
export interface AddressInputModel {
    userId: number;
    cep: string;
    street: string;
    streetNumber: string;
    complement?: string | null;
    neighborhood: string;
    city: string;
    state: string;
}
```